developer/src/kmc-copy/src/KeymanProjectCopier.ts

```diff
@@ -15,6 +15,9 @@ type CopierFunction = (
   project: KeymanDeveloperProject, filename: string, outputPath: string, source: string, result: CopierResult
 ) => Promise<boolean>;
 
+const KEYMANCOM_CLOUD_URI = /^(?:http(?:s)?:\/\/)?keyman\.com\/keyboards\/(?<id>[a-z0-9_.-]+)/i;
+const CLOUD_URI = /^cloud:(?<id>.+)$/i;
+
```

Comment:
It's only a few lines, but since GitHubUrls got refactored, thoughts on pulling these into a separate `cloud-uri.ts file`? (along with some comments about the regexes)